### ISSUE TITLE
DropDownTree accessibility - Certain ARIA roles must contain particular children

### ISSUE THREAD

**Issue by Dimitar-Goshev** [telerik/kendo-ui-core#6558]
### Bug report
DropDownTree accessibility issue - Certain ARIA roles must contain particular children

### Reproduction of the problem
1. Go to https://demos.telerik.com/kendo-ui/dropdowntree/client-filtering
2. Run Axe DevTools or Accessibility Insights for Web in Chrome

### Current behavior
Certain ARIA roles must contain particular children 

Element location - .k-dropdowntree
Element source
`<span title="" class="k-widget k-dropdowntree k-dropdowntree-clearable" unselectable="on" role="listbox" aria-haspopup="true" aria-expanded="false" tabindex="0" aria-disabled="false" aria-readonly="false" style="width: 100%;">`
To solve this issue, you need to fix the following: Required ARIA child role not present: option

### Expected/desired behavior
There should be elements with role option associated to the element with role listbox

### Environment
* **Kendo UI version:** 2021.2.616
* **Browser:** all
